src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String RESULT_FILE_FIELDS = "fruit,quantity\n";
```

Comment:
use `System.lineSeparator()` instead of `\n`
```suggestion
    private static final String HEADER = "fruit,quantity";
```